themes/sgsc/sass/_contact-us.scss

```diff
@@ -34,25 +34,37 @@
   display: flex;
   flex-flow: row wrap;
   justify-content: space-between;
+  padding: $margin-d;  
+  margin-top: 0px;
   @media screen and (min-width: $tablet_width) {
     margin-bottom: 50px;
   }
   .contactus {
-    padding: $margin-d;
     width: 100%;
     @media screen and (min-width: $mobile_width) {
       width: 100%;
+      max-height: 300px;
+      display: flex;
+      overflow: hidden;
+      flex-flow: column wrap;
+      p {
+        margin: 18px;
```

Comment:
This margin makes it so that the text doesn't line up with the map at tablet size... I think we can do without it!